docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/irsa.md

```diff
@@ -102,135 +102,84 @@ IAM Roles for Service Accounts can only be implemented with Keycloak 21 onwards.
-    implementation group: 'software.amazon.awssdk', name: 'metrics-spi', version: awsSdkVersion
-    implementation group: 'software.amazon.awssdk', name: 'profiles', version: awsSdkVersion
-    implementation group: 'software.amazon.awssdk', name: 'protocol-core', version: awsSdkVersion
-    implementation group: 'software.amazon.awssdk', name: 'rds', version: awsSdkVersion
-    implementation group: 'software.amazon.awssdk', name: 'regions', version: awsSdkVersion
-    implementation group: 'software.amazon.awssdk', name: 'sdk-core', version: awsSdkVersion
-    implementation group: 'software.amazon.awssdk', name: 'sts', version: awsSdkVersion
-    implementation group: 'software.amazon.awssdk', name: 'third-party-jackson-core', version: awsSdkVersion
-    implementation group: 'software.amazon.awssdk', name: 'utils', version: awsSdkVersion
-}
-
-task copyDependencies(type: Copy) {
-    from configurations.runtimeClasspath
-    into "lib"
-}
-```
-
-Create a file called `Dockerfile` with the following content in the same directory:
-
-```shell
-FROM gradle:jdk17-focal as lib
-
-WORKDIR /home/gradle
-
-COPY build.gradle /home/gradle
-
-RUN gradle copyDependencies
-
-FROM keycloak/keycloak:22.0 as builder
-
-# Enable health and metrics support
-ENV KC_HEALTH_ENABLED=true
-ENV KC_METRICS_ENABLED=true
-
-# Configure a database vendor
-ENV KC_DB=postgres
-
-COPY --from=lib /home/gradle/lib /opt/keycloak/providers
-
-WORKDIR /opt/keycloak
+#### Kubernetes configuration
 
-RUN /opt/keycloak/bin/kc.sh build
+As an example, configure the following environment variables to enable IRSA:
 
-FROM keycloak/keycloak:22.0
+```yaml
+# The AWS wrapper is not capable of XA transactions
+- name: KEYCLOAK_EXTRA_ARGS
+  value: "--db-driver=software.amazon.jdbc.Driver --transaction-xa-enabled=false --log-level=INFO,software.amazon.jdbc:INFO"
 
-COPY --from=builder /opt/keycloak/ /opt/keycloak/
+# Enable the AWS IAM plugin
+- name: KEYCLOAK_JDBC_PARAMS
+  value: "wrapperPlugins=iam"
+- name: KEYCLOAK_JDBC_DRIVER
+  value: "aws-wrapper:postgresql"
 
-ENV KC_DB=postgres
+# Configure database
+- name: KEYCLOAK_DATABASE_USER
+  value: db-user-name
+- name: KEYCLOAK_DATABASE_NAME
+  value: db-name
+- name: KEYCLOAK_DATABASE_HOST
+  value: db-host
+- name: KEYCLOAK_DATABASE_PORT
+  value: 5432
+
+# Ref: https://www.keycloak.org/server/configuration-metrics
+- name: KEYCLOAK_ENABLE_STATISTICS
+  value: "true"
 
-ENTRYPOINT ["/opt/keycloak/bin/kc.sh"]
+# Needed to see if Keycloak is healthy: https://www.keycloak.org/server/health
+- name: KEYCLOAK_ENABLE_HEALTH_ENDPOINTS
+  value: "true"
 ```
 
-Run the following command or variation with `docker buildx` to create and publish the required image.
-
-```
-docker build . --no-cache
-```
+:::note
+Don't forget to set the `serviceAccountName` of the deployment/statefulset to the created service account with the IRSA annotation.
+:::
 
-#### Kubernetes configuration
+##### Helm chart
```

Comment:
Note that H4 and H5 headers will render equal to, if not slightly smaller than the standard text.

Would love to avoid this if possible, but will also circle back to it in https://github.com/camunda/developer-experience/issues/257.